web_src/less/_repository.less

```diff
@@ -675,9 +672,17 @@
                     margin: .3em 0 .5em .5em;
                 }
 
+                :nth-child(3) {
+                    margin-left: auto;
+                }
```

Comment:
Maybe introduce additional divs like `.review-item-left` and `.review-item-right` and do `display: flex; justify-content: space-between` on the parent?

Reply:
Would need to set `display: flex` on these divs too because their children would be just `inline-block` which breaks alignment.

Reply:
```patch
Index: web_src/less/_repository.less
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- web_src/less/_repository.less	(revision 2f55aa080f1471194b1faa6cfc02adf1fafd20bc)
+++ web_src/less/_repository.less	(date 1592064389896)
@@ -658,8 +658,14 @@
 
             .review-item {
                 display: flex;
+                justify-content: space-between;
                 align-items: center;
 
+                .review-item-left,
+                .review-item-right {
+                    display: flex;
+                }
+
                 .avatar,
                 .type-icon {
                     .svg {
@@ -672,10 +678,6 @@
                     margin: .3em 0 .5em .5em;
                 }
 
-                :nth-child(3) {
-                    margin-left: auto;
-                }
-
                 .type-icon {
                     align-self: flex-start;
                     margin-right: 1em;
```

Reply:
Maybe also set `align-items: center` on both `.review-item` and the left/right children if that helps any.

Reply:
Doesn't do anything actually but can add it.